moped-editor/src/components/Maps/GeocoderControl.js

```diff
@@ -0,0 +1,143 @@
+import * as React from "react";
+import { useState } from "react";
+import { useControl, Marker } from "react-map-gl";
+import MapboxGeocoder from "@mapbox/mapbox-gl-geocoder";
+
+// See MOPED Technical Docs > Dev Guides > Maps and geospatial data  > Libraries
+const austinFullPurposeJurisdictionFeatureCollection = {
+  type: "FeatureCollection",
+  crs: {
+    type: "name",
+    properties: {
+      name: "EPSG:4326",
+    },
+  },
+  bbox: [-97.940377, 30.133717, -97.578205, 30.464826],
+  features: [],
+};
+
+/* eslint-disable complexity,max-statements */
+// See https://github.com/visgl/react-map-gl/tree/7.0-release/examples/geocoder
```

Comment:
This entire file was copied and de-Typscriptified straight from the `react-map-gl` code example (with small mods like the bounding box to keep search results in the Austin Full Purpose Jurisdiction and to turn off showing a mark on the location selected).